src/api/app/controllers/concerns/webui/requests_filter.rb

```diff
@@ -5,9 +5,9 @@ def filter_by_involvement(requests, filter_involvement)
     return requests if filter_involvement == 'all'
 
     if filter_involvement == 'incoming'
-      User.session.incoming_requests
+      requests.where(User.session.incoming_requests)
     elsif filter_involvement == 'outgoing'
-      User.session.outgoing_requests
+      requests.where(User.session.outgoing_requests)
```

Comment:
```suggestion
      requests.where(id: User.session.outgoing_requests)
```
I realized this was failiing when working on #16540. I added the fixed code there, as a separate commit.

Reply:
I don't think so.